backend-api/template.yaml

```diff
@@ -428,7 +428,6 @@ Resources:
                     dynamodb:Select: SPECIFIC_ATTRIBUTES
               - Effect: Allow
                 Action:
-                  - dynamodb:GetItem
```

Comment:
Getting the session before creating it is no longer required. We can use a condition expression to ensure that a new session is only created if one with the same session ID does not already exist.